tests/js/client/shell/api/multi/general-graph.js

```diff
@@ -1257,179 +1298,195 @@ function check400CRUD (doc) {
 
 // describe "should throw 404 if document is unknown on route" do;
 
-function check404 (doc) {
+function check404(doc) {
   assertEqual(doc.code, internal.errors.ERROR_HTTP_NOT_FOUND.code, doc);
   assertTrue(doc.parsedBody['error'], doc);
   assertEqual(doc.parsedBody['code'], internal.errors.ERROR_HTTP_NOT_FOUND.code, doc);
   assertEqual(doc.parsedBody['errorNum'], internal.errors.ERROR_ARANGO_DOCUMENT_NOT_FOUND.code, doc);
   assertMatch(/.*document not found.*/, doc.parsedBody['errorMessage'], doc);
 }
 
-function check404Collection (doc) {
+function check404Collection(doc) {
   assertEqual(doc.code, internal.errors.ERROR_HTTP_NOT_FOUND.code);
   assertTrue(doc.parsedBody['error']);
   assertEqual(doc.parsedBody['code'], internal.errors.ERROR_HTTP_NOT_FOUND.code);
   assertEqual(doc.parsedBody['errorNum'], internal.errors.ERROR_ARANGO_DATA_SOURCE_NOT_FOUND.code);
   assertMatch(/.*collection or view not found.*/, doc.parsedBody['errorMessage'], doc);
 }
 
-function check400Collection (doc) {
+function check400Collection(doc) {
   assertEqual(doc.code, internal.errors.ERROR_HTTP_BAD_PARAMETER.code);
   assertTrue(doc.parsedBody['error']);
   assertEqual(doc.parsedBody['code'], internal.errors.ERROR_HTTP_BAD_PARAMETER.code);
   assertEqual(doc.parsedBody['errorNum'], internal.errors.ERROR_ARANGO_DATA_SOURCE_NOT_FOUND.code);
   assertMatch(/.*no collection name specified.*/, doc.parsedBody['errorMessage'], doc);
 }
 
-function check400_edge_missing (doc) {
+function check400_edge_missing(doc) {
   assertEqual(doc.code, internal.errors.ERROR_HTTP_BAD_PARAMETER.code);
   assertTrue(doc.parsedBody['error']);
   assertEqual(doc.parsedBody['code'], internal.errors.ERROR_HTTP_BAD_PARAMETER.code);
   assertEqual(doc.parsedBody['errorNum'], internal.errors.ERROR_ARANGO_INVALID_EDGE_ATTRIBUTE.code);
   assertMatch(/.*expecting both `_from` and `_to` attributes to be defined.*/, doc.parsedBody['errorMessage'], doc);
 }
 
-function check404_graph_not_found (doc) {
+function check404_graph_not_found(doc) {
   assertEqual(doc.code, internal.errors.ERROR_HTTP_NOT_FOUND.code);
   assertTrue(doc.parsedBody['error']);
   assertEqual(doc.parsedBody['code'], internal.errors.ERROR_HTTP_NOT_FOUND.code);
   assertEqual(doc.parsedBody['errorNum'], internal.errors.ERROR_GRAPH_NOT_FOUND.code);
   assertEqual(doc.parsedBody['errorMessage'], "graph 'UnitTestUnknown' not found");
+}
+
+const assertVertexCollectionNotKnown = (response) => {
+  assertTrue(response.parsedBody);
```

Comment:
This assertion does not make sense to me as a reader of the code. Did you mean `assertTrue(response.hasOwnProperty('parsedBody'));`?

Reply:
Ah, yes - better! Thanks for spotting. This has been just a copy-paste. Will replace all occurrences with your suggestion.